electron-app/src/server/websites/bluesky/bluesky.service.ts

```diff
@@ -182,7 +182,9 @@ export class Bluesky extends Website {
 
   formatTags(tags: string[]) {
     return this.parseTags(
-      tags.map(tag => tag.replace(/[^a-z0-9]/gi, ' ')).map(tag => tag.split(' ').join('')),
+      tags
+        .map(tag => tag.replace(/[^\p{Letter}\p{Number}]/giu, ' '))
```

Comment:
q: is this a newer js feature? (and by newer I mean anything after 2018 since I haven't been able to keep up with all the new specs)

Reply:
Uh, I didn't check, Perl had this since 2000 so I'd assumed that everyone that copied their regular expressions from there had it too. Looking it up, apparently it was introduced into JavaScript in 2015, so I guess they were 15 years late to the party, but still have had it for a while now.